src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java

```diff
@@ -18,6 +18,7 @@
 public class NotificationValidService {
     private static final List<NotificationEventType> MANDATORY_LETTER_EVENT_TYPES = Arrays.asList(STRUCK_OUT, HEARING_BOOKED_NOTIFICATION);
```

Comment:
Just a small readability suggestion - keep names in alphabetical order. It is not an issue here as the lists are small.